audit-service/README.md

```diff
@@ -38,27 +61,43 @@ been implemented. This service does not have any output Kafka topics.
 
 *The token value come from the headers of the Kafka message that the service receives. This links the audit message to the original request that was made.
 
-If an error has occurred during a request/read then an AuditErrorMessage will be consumed from the `error` Kafka topic. This type of message can be sent from any of the Palisade services
-(e.g user-service or policy-service).
+If an error has occurred while Palisade is processing a client request then an AuditErrorMessage will be added to the `error` Kafka topic by the service that encountered the issue. 
+This type of message can be sent from all the Palisade services.
```

Comment:
```suggestion
This type of message can be sent from any of the Palisade services.
```

Reply:
This was changed based on previous comments but I do think this works better. Will change back.